src/components/Header.vue

```diff
@@ -30,6 +26,10 @@
   function addToDo() {
     emit('toDoAdded');
   }
+  function searchToDo(searchField) {
+    console.log(searchField);
```

Comment:
Please add descriptive log message here, not just the value, or remove this line.